library/src/interactor_impl.cxx

```diff
@@ -622,8 +632,8 @@ interactor& interactor_impl::initCommands()
   {
     if (args.size() != expectedSize)
     {
-      throw interactor_impl::invalid_args_exception(std::string("Command: ") +
-        std::string(actionName) + " is expecting " + std::to_string(expectedSize) + " arguments");
+      throw interactor::invalid_args_exception(std::string("Command: ") + std::string(actionName) +
```

Comment:
Do we even need to add the scope of the exception here (and everywhere)?

Reply:
Not technically needed but this is more understandable this way. I've been doing that for exceptions everywhere in the code.